data/bash-completion/fwupdmgr

```diff
@@ -56,7 +56,6 @@ _fwupdmgr_cmd_list=(
 
 _fwupdmgr_opts=(
 	'--verbose'
-	'--offline'
```

Comment:
Shouldn't this be dependant on whether the option is set? Can we even do that?

Reply:
fixed